contribute/design/1.x/migrate/spc-to-cspc-migration.md

```diff
@@ -0,0 +1,228 @@
+
+The import command will be modified to import with or without the oldname. For example for renaming the command would look like:
+```sh
+/usr/local/bin/zpool import  -o cachefile=/var/openebs/cstor-pool/pool.cache  -d /var/openebs/sparse cstor-08bfced5-6a28-4a63-a76b-c48c69be6ad5 cstor-6b3bbf9e-451d-4333-b119-1bb5217e5bc2
+```
+For importing without renaming the command would look like:
+```sh
+/usr/local/bin/zpool import  -o cachefile=/var/openebs/cstor-pool/pool.cache  -d /var/openebs/sparse cstor-6b3bbf9e-451d-4333-b119-1bb5217e5bc2
+```
+
+
+The `cstorpoolinstance.openebs.io/oldname` annotation will be used to rename the pool which was named as `cstor-cspuid` to `cstor-cspcuid`. This annotation will be removed after successful import of the pool.
+
+**Note: Before migrating make sure:**
+ - OpenEBS version of old resources(control-plane and data-plane) should be atleast 1.11.0.
+ - Apply new cstor-operators and csi driver which also should be atleast in 1.11.0 version.
+ - Identify the nodes on which the `CSP` are present and run `fdisk -l` command on that node. If the command is hung then bad block file exists on that node. If such case occurs, please resolve this issue before proceeding with the migration. To identify the nodes look for the `kubernetes.io/hostname` label on the `CSP`.
+
+
+The Job spec for migrating SPC is:
+```yaml
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: migrate-spc-cstor-sparse-pool
+  namespace: openebs
+spec:
+  backoffLimit: 4
+  template:
+    spec:
+      #VERIFY the value of serviceAccountName is pointing to service account
+      # created within openebs namespace. Use the non-default account.
+      # by running `kubectl get sa -n <openebs-namespace>`
+      serviceAccountName: openebs-maya-operator
+      containers:
+      - name:  migrate
+        args:
+        - "cstor-spc"
+        # name of the spc that is to be migrated
+        - "--spc-name=cstor-sparse-pool"
+
+        #Following are optional parameters
+        #Log Level
+        - "--v=4"
+        #DO NOT CHANGE BELOW PARAMETERS
+        env:
+        - name: OPENEBS_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        tty: true
+
+        image: quay.io/openebs/migrate:ci
+      restartPolicy: OnFailure
+---
+```
+
+After successful completion of pool migration the logs will list out all applications having volumes on the given pool.
+
+#### Phase 2: Migration of Non-CSI volume to CSI volume
+
+**Note: Before proceeding to the below steps the pool must be migrated successfully and all the applications having volumes on the pool must be scale down.**
```

Comment:
Why all the applications only application consuming that corresponding pool should be scale down right?